src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -2166,6 +2187,7 @@ private static TypeSymbolWithAnnotations InferResultNullability(BoundExpression
                             break;
                         }
                         // PROTOTYPE(NullableReferenceTypes): Update method based on operandType.
```

Comment:
Is this comment still applicable?

Reply:
Yes. Added a test that fails currently: `StaticNullChecking_FlowAnalysis.Conversions_07`.

---
In reply to: [183807511](https://github.com/dotnet/roslyn/pull/26356#discussion_r183807511) [](ancestors = 183807511)